services/QuillLMS/client/package.json

```diff
@@ -151,7 +151,7 @@
     "@types/jest": "^22.2.0",
     "@wojtekmaj/enzyme-adapter-react-17": "^0.6.6",
     "babel-jest": "^29.7.0",
-    "cheerio": "^0.22.0",
+    "cheerio": "1.0.0-rc.4",
```

Comment:
I think we should avoid release-candidates for production code.  I would go for 1.1.2 if possible: https://www.npmjs.com/package/cheerio?activeTab=versions

Reply:
<img width="812" height="490" alt="image" src="https://github.com/user-attachments/assets/10937952-5b6b-4c7c-b5fa-d625b5dbef75" />

Discussed with Eric offline.  Normally, release candidates should not be used in production, but rc12 looks like the conventional choice here.